### PR TITLE
Makefile: Fix clean target to delete sub-package coverage data

### PR DESCRIPTION
**Pull request by mmussomele**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -31,8 +31,8 @@ check: format-check
 
 clean:
 	go clean -x $(PACKAGES)
-	rm -f *.cov.coverprofile
-	rm -f *.cov.html
+	rm -f *.cov.coverprofile cluster/*.cov.coverprofile minion/*.cov.coverprofile specs/*.cov.coverprofile
```
Comment (kklin) [inline Makefile]: Maybe just using a `find` command would be better? Otherwise these directories will need to be kept up to date.

Comment (mmussomele) [inline Makefile]: I would, but `find . -name "*.cov.coverprofile" -delete` ins't supported on all platforms

Comment (mmussomele) [inline Makefile]: although I suppose `find . -name "*.cov.coverprofile" -exec rm -f {} \;` should work too

**Comment by mmussomele** [comment]
merged

**Pull request closed without merging**